site/en/community/users.md

```diff
@@ -251,6 +251,30 @@ for our go, java and python projects.
 LinkedIn, a subsidiary of Microsoft, is the world’s largest professional social
 network. LinkedIn uses Bazel for building its iOS Apps.
 
+### [Lucid Software](https://lucid.co/){: .external}
+
+<!-- 
+TODO: Only for the internal PR review. 
+Remove when submitting the final PR would need the /community/ base path,
+which would eventually become /static/community/ on the
+served site https://bazel.build/community/users#companies-using-bazel
+-->
+<img src="images/Lucid_Software-logo.svg" width="150" align="right">
+
+<!-- 
+TODO: Uncomment after internal PR review.
+<img src="/community/images/Lucid_Software-logo.svg" width="150" align="right">
+-->
+
+Lucid is the leader in visual collaboration for enterprise and hybrid teams of
+all sizes. We help teams see and build the future.
+
+Lucid uses Bazel to build millions of lines of Scala and TypeScript
```

Comment:
misssing period after TypeScript (note: markdown doesn't respect implicit newlines)

Reply:
Fixed

Reply:
(reply to note: Yes, that was a formatting typo)